main.nf

```diff
@@ -408,6 +431,7 @@ workflow {
     FEATURE_ENGINEERING_PART2 (
         FEATURE_ENGINEERING_PART1.out[0],
         PHRANK_SCORING.out,
+        NORMALIZE_NETWORK.out,
```

Comment:
I thought we don’t need to rerun the process once we create. Isn’t it?

Reply:
What do you mean?